water_monitoring_system/config/settings/local.py

```diff
@@ -0,0 +1,60 @@
+from .base import *  # noqa
+from .base import env
+
+# GENERAL
+# ------------------------------------------------------------------------------
+# https://docs.djangoproject.com/en/dev/ref/settings/#debug
+DEBUG = True
+# https://docs.djangoproject.com/en/dev/ref/settings/#secret-key
+SECRET_KEY = env(
+    "DJANGO_SECRET_KEY",
+    default="Z4XJyzqqR1FbNiak02bKbcqYEQGOGPwrfuRZpDxxfkZtToJBYstQrGE3E189TSpU",
```

Comment:
Is this key sensitive?
If yes, please remove.

Reply:
No this is for local use only,
for production we will change the key